package.js

```diff
@@ -121,3 +121,7 @@ Package.onTest(function (api) {
   // limited to only working on the server
   // api.addFiles('tests/fetch.js', 'server')
 })
+
+Npm.depends({
```

Comment:
Shouldn't we make this a peer dependency and use 
https://github.com/tmeasday/check-npm-versions

I think this is the recommended way

Reply:
tbh i didn't know that - will update the pr in a few minutes

Reply:
@namirsab i can't get this to work inside the tests:
```
diff --git a/collection-hooks.js b/collection-hooks.js
index 717a167..ef12496 100644
--- a/collection-hooks.js
+++ b/collection-hooks.js
@@ -1,6 +1,9 @@
-/* global Package Meteor Mongo LocalCollection CollectionHooks _ EJSON */
+import { checkNpmVersions } from 'meteor/tmeasday:check-npm-versions';
 /* eslint-disable no-proto, no-native-reassign, no-global-assign */
 
+checkNpmVersions({
+  'lodash': '4.17.11'
+}, 'matb33:collection-hooks');
 // Relevant AOP terminology:
 // Aspect: User code that runs before/after (hook)
 // Advice: Wrapper code that knows when to call user code (aspects)
@@ -9,11 +12,11 @@ var isFunction = require('lodash/isFunction')
 var toArray = require('lodash/toArray')
 var omit = require('lodash/omit')
 
-
 var advices = {}
 var Tracker = (Package.tracker && Package.tracker.Tracker) || Package.deps.Deps
 var publishUserId = Meteor.isServer && new Meteor.EnvironmentVariable()
 
+/* global Package Meteor Mongo LocalCollection CollectionHooks EJSON */
 CollectionHooks = {
   defaults: {
     before: {insert: {}, update: {}, remove: {}, upsert: {}, find: {}, findOne: {}, all: {}},
diff --git a/package.js b/package.js
index 822ee15..5d70e84 100644
--- a/package.js
+++ b/package.js
@@ -30,7 +30,8 @@ Package.onUse(function (api, where) {
   api.use([
     'ejson',
     'minimongo',
-    'ecmascript'
+    'ecmascript',
+    'tmeasday:check-npm-versions'
   ])
 
   api.use(['accounts-base'], ['client', 'server'], {weak: true})
@@ -120,8 +121,4 @@ Package.onTest(function (api) {
   // NOTE: fetch can only work server-side because find's 'fields' option is
   // limited to only working on the server
   // api.addFiles('tests/fetch.js', 'server')
-})
-
-Npm.depends({
-  'lodash': '4.17.10'
-});
\ No newline at end of file
+})
\ No newline at end of file
diff --git a/package.json b/package.json
index 91a361a..db3e4cb 100644
--- a/package.json
+++ b/package.json
@@ -28,6 +28,7 @@
     "eslint-plugin-promise": "^3.5.0",
     "eslint-plugin-react": "^7.0.1",
     "eslint-plugin-standard": "^3.0.1",
-    "spacejam": "^1.6.1"
+    "spacejam": "^1.6.1",
+    "lodash": "^4.17.11"
   }
 }
```
it seems to work fine in dev, but the tests fail with:
```
W20180913-12:41:26.796(2)? (STDERR) /home/lukass/.meteor/packages/meteor-tool/.1.7.0_5.r5qyn1.b1gbd++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
W20180913-12:41:26.797(2)? (STDERR)                                             throw(ex);
W20180913-12:41:26.797(2)? (STDERR)                                             ^
W20180913-12:41:26.797(2)? (STDERR) 
W20180913-12:41:26.797(2)? (STDERR) Error: lodash@4.17.11 not installed.
W20180913-12:41:26.797(2)? (STDERR)     at failures.forEach.name (packages/tmeasday:check-npm-versions/check-npm-versions.js:28:11)
W20180913-12:41:26.798(2)? (STDERR)     at Array.forEach (<anonymous>)
W20180913-12:41:26.798(2)? (STDERR)     at checkNpmVersions (packages/tmeasday:check-npm-versions/check-npm-versions.js:26:12)
W20180913-12:41:26.798(2)? (STDERR)     at collection-hooks.js (packages/matb33:collection-hooks/collection-hooks.js:4:1)
W20180913-12:41:26.798(2)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:322:7)
```

any idea?

Reply:
hm @tmeasday maybe can help here.

Reply:
Can't remember how this stuff works, something about how the test app is a clone of your real app in a temp directory. Sorry!

Reply:
I think it would be better to leave out lodash altogether, I think there are reasonable vanilla-js alternatives for every used function.